src/Elastic.Markdown/Myst/CodeBlocks/SupportedLanguages.cs

```diff
@@ -0,0 +1,193 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+namespace Elastic.Markdown.Myst.CodeBlocks;
+
+public static class CodeBlock
+{
+	private static readonly IReadOnlyDictionary<string, string> LanguageMapping = new Dictionary<string, string>
```

Comment:
Where is this list coming from? Is it https://github.com/highlightjs/highlight.js/blob/main/SUPPORTED_LANGUAGES.md (excluding the extra packages)?

Reply:
Exactly that :)